src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorFactory.cs

```diff
@@ -158,18 +234,21 @@ public static class TagHelperDescriptorFactory
             string assemblyName,
             IEnumerable<TagHelperAttributeDescriptor> attributeDescriptors,
             IEnumerable<string> requiredAttributes,
+            IEnumerable<string> allowedChildren,
             TagStructure tagStructure,
             TagHelperDesignTimeDescriptor designTimeDescriptor)
         {
-            return new TagHelperDescriptor(
-                prefix: string.Empty,
-                tagName: tagName,
-                typeName: typeName,
-                assemblyName: assemblyName,
-                attributes: attributeDescriptors,
-                requiredAttributes: requiredAttributes,
-                tagStructure: tagStructure,
-                designTimeDescriptor: designTimeDescriptor);
+            return new TagHelperDescriptor
+            {
+                TagName = tagName,
+                TypeName = typeName,
+                AssemblyName = assemblyName,
+                Attributes = attributeDescriptors,
+                RequiredAttributes = requiredAttributes,
+                AllowedChildren = allowedChildren,
+                TagStructure = tagStructure,
+                DesignTimeDescriptor = designTimeDescriptor
```

Comment:
Side note: Not for this PR but please include the final comma when using multi-line initializers. Results in less noise in PRs that add / remove properties later.